apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -387,7 +387,11 @@ def suspended?
     # Whether job is running but still hasn't generated the connection file
     # @return [Boolean] whether starting
     def starting?
-      status.running? && !connect_file.file?
+      if native_connection_info?
+        status.queued? && !connect.to_h.compact.empty?
```

Comment:
[Carry over from other commit's conversation]

I thought about this more, and I think `queued` is the right status here for k8s.  If we call a pod `running` when it's init containers are running, it's not actually available.  There could be some long running init container. Even if  `connect.to_h.compact >= 3`, it doesn't guarantee _some_ init container is still running. 

This expression can differentiate between init containers are running (queued status + non empty connect info) which is logically 'starting' and an unschedulable pod (queued status, but you'd have an empty empty connect info, after compact, because there's no host info yet).